main.go

```diff
@@ -0,0 +1,8 @@
+package main
+
+import (
+	"github.com/cilium/customvet/analysis"
+	"golang.org/x/tools/go/analysis/singlechecker"
```

Comment:
I imagine we want to adhere to the same code style guidelines as the main Cilium codebase so import ordering here should be:

```
import (
  std

  cilium/customvet

  3rd party
)
```